src/web-ui/src/public/Welcome.vue

```diff
@@ -32,7 +32,7 @@
           <div class="login-cta d-flex justify-content-center align-items-center">
             Have an account?<router-link to="/auth" class="sign-in btn btn-link">Sign in</router-link>
           </div>
-          <router-link to="/auth" class="create-account mt-3 mt-sm-0 ml-sm-3 btn btn-primary"
+          <router-link to="/auth?signup=true" class="create-account mt-3 mt-sm-0 ml-sm-3 btn btn-primary"
```

Comment:
Since we've made the change the comment on line 29 is talking about, let's remove that comment as well.